.haml-lint.yml

```diff
@@ -10,21 +10,24 @@ linters:
     enabled: true
 
   ClassesBeforeIds:
-    enabled: true
+    enabled: false
 
   ConsecutiveComments:
     enabled: true
 
   ConsecutiveSilentScripts:
     enabled: true
-    max_consecutive: 2
+    max_consecutive: 5
 
   EmptyScript:
     enabled: true
 
   HtmlAttributes:
     enabled: true
 
+  IdNames:
```

Comment:
could you elaborate what this attribute is for?

Reply:
I think you got to it just before I pushed my commit-- this is what I added:

```haml
# E.g. using lispCase instead of snake_case
```